server/mldb_server.cc

```diff
@@ -263,6 +271,32 @@ runHttpQuery(const Utf8String& query,
                        rowNames, rowHashes, sortColumns);
 }
 
+void
+MldbServer::
+handleRedirectToGet(const string & uri,
+                    RestConnection & connection,
+                    const Json::Value & body) const
+{
```

Comment:
Would checking that the target starts with `/v1` be enough to block such attempts?